autocertkit/utils.py

```diff
@@ -1844,12 +1844,19 @@ def init_vifs_ip_addressing(session, vm_ref, vifs_info):
                       (vm_ref, device))
 
 
+def modify_interface_rp_filter(mip, iface):
```

Comment:
I think the name could be more specific. "Modify" is quite general. What is `rp_filter=0` doing? This also goes for the comment in the code. What exactly does this? That this is writing to a file is obvious but not what is accomplishes.

I don't see the need for `-e`; an `echo` without `-e` and `\n` would do the same thing but if this how it's done consistently in this code base it's fine.

Is writing to `/etc/sysctl.d/99-customer.conf` enough for it to take effect? Or does this require some form of reload? Has this been tested?